.github/workflows/test.yml

```diff
@@ -0,0 +1,58 @@
+name: Run Tests
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    # Run automatically at 8AM PST Monday-Friday
+    - cron: '0 15 * * 1-5'
+  workflow_dispatch:
+
+jobs:
+  tests:
+    name: Build & Test
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    strategy:
+      matrix:
+        go: [ '1.11.x', '1.12.x', '1.13.x', '1.14.x', '1.15.x', '1.16.x' ]
+    steps:
+      - name: Checkout smtpapi-go
+        uses: actions/checkout@v2
+
+      - name: Setup Go environment
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go }}
+
+      - name: Set GOPATH
+        run: echo "GOPATH=$HOME" >> $GITHUB_ENV
+      - name: Set GOBIN
+        run: echo "GOBIN=$GOPATH/bin" >> $GITHUB_ENV
+      - name: Set GO111MODULE
+        run: echo "GO111MODULE=off" >> $GITHUB_ENV
+
+      - name: Build smtpapi-go
+        run: make install
+
+      - name: Run Tests
+        run: make test
+
+  notify-on-failure:
+    name: Slack notify on failure
+    if: ${{ failure() && github.ref == 'refs/heads/main' && github.event_name != 'pull_request' }}
+    needs: [ tests ]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_COLOR: 'danger'
+          SLACK_ICON_EMOJI: ':github:'
+          SLACK_MESSAGE: ${{ format('Failed running build on {1}{3} {0}/{1}/actions/runs/{2}', github.server_url, github.repository, github.run_id, ':') }}
+          SLACK_TITLE: Build Failure
+          SLACK_USERNAME: GitHub Actions
+          SLACK_MSG_AUTHOR: twilio-dx
+          SLACK_FOOTER: Posted automatically using GitHub Actions
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+          MSG_MINIMAL: true
```

Comment:
nit: newline

Reply:
Should be an option in your IDE to always end a file with a newline.